Italian Authentic Sources' Onboarding and EAA design Handbook (#1063)
italia/eidas-it-wallet-docs: handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md

## Indice dei contenuti

- [Introduzione e contesto](#Introduzione-e-contesto)
- [Scopo e ambito di applicazione](#Scopo-e-ambito-di-applicazione)
- [Ruoli e responsabilità del Titolare di Fonte Autentica](#Ruoli-e-responsabilità-del-Titolare-di-Fonte-Autentica)
- [Come diventare Titolare di Fonte Autentica](#Come-diventare-Titolare-di-Fonte-Autentica)
  - [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa) 
  - [Step 2 | Pubblicazione in collaudo](#step-2--pubblicazione-in-collaudo) 
  - [Step 3 | Test in collaudo](#step-3--test-in-collaudo)
  - [Step 4 | Pubblicazione in produzione](#step-4--pubblicazione-in-produzione) 
  - [Step 5 | Test in produzione](#step-5--test-in-produzione)
  - [Step 6 | Pianificazione rilascio EAA](#step-6--pianificazione-rilascio-eaa)
  - [Step 7 | Manutenzione e assistenza](#step-7--manutenzione-e-assistenza)
- [File da compilare](#file-da-compilare)
- [Appendice A – Casi d'uso](#appendice-a--casi-duso)
- [Appendice B – Data Model](#appendice-b--data-model)
- [Appendice C – Mappatura errori](#appendice-c--mappatura-errori)

> - [Appendice D – Mappatura stati](#appendice-c--mappatura-stati)

Reviewer: ```suggestion
- [Appendice D – Mappatura stati](#appendice-d--mappatura-stati)
```
typo

- [Appendice E – Assistenza](#appendice-e--assistenza)

> - [Template PDND Data Model](#Template-PDND-Data-Model)

Reviewer: ```suggestion
- [Template e-service PDND](#Template-eservice-PDND)
```
Template name fixed

  [Vai allo Step 2](#step-2--pubblicazione-in-collaudo)
- **Step 3 | Test in collaudo**: l'Ente, in ambiente di collaudo PDND, può eseguire i test di integrazione dell'e-service e di Signal Hub con il Fornitore di Attestati Elettronici di Attributi indicato come fruitore e, se ritenuto necessario, con il Fornitore di Wallet, anche gli aspetti relativi alla UX/UI dell’EAA. 
  [Vai allo Step 3](#step-3--test-in-collaudo)
- **Step 4 | Pubblicazione in produzione**: l'Ente rilascia l'e-service in ambiente di produzione su PDND e attiva il relativo servizio Signal Hub in produzione, al fine di supportare una corretta gestione del ciclo di vita dell’EAA. Infine, l'Ente notifica al Fornitore di Attestati Elettronici di Attributi l'avvenuta pubblicazione.
  [Vai allo Step 4](#step-4--pubblicazione-in-produzione)

> - **Step 5 | Test in produzione**: l'Ente, in ambiente di produzione, esegue i test di integrazione, di carico e long run dell'e-service. Per i dettagli implementativi, consultare le Specifiche Tecniche italiane:

Reviewer: ```suggestion
- **Step 5 | Test in produzione**: l'Ente, in ambiente di produzione, esegue i test di integrazione, di carico e long run dell'e-service con il Fornitore di Attestati Elettronici di Attributi e, ove possibile se richiesto, con il Fornitore di Wallet per testare anche gli aspetti relativi alla UX/UI.
```

  [Vai allo Step 5](#step-5--test-in-produzione)

> - **Step 6 | Pianificazione rilascio EAA**: a valle del buon esito dei test in collaudo e in produzione, l'Ente concorda con il Fornitore di Attestati Elettronici di Attributi la data di rilascio, per l'ottenimento dell’EAA da parte degli utenti. Inoltre, l’Ente può valutare attività di comunicazione, in sinergia con gli altri attori interessati.

Reviewer: ```suggestion
- **Step 6 | Pianificazione rilascio EAA**: a valle del buon esito dei test in collaudo e in produzione, l'Ente concorda con il Fornitore di Attestati Elettronici di Attributi, e se possibile con il Fornitore di Wallet, la data di rilascio, per l'ottenimento dell’EAA da parte degli utenti. Inoltre, l’Ente può valutare attività di comunicazione, in sinergia con gli altri attori interessati.
```

- **orientare il tipo di formato** con cui il Fornitore di Attestati Elettronici emetterà l'EAA (SD-JWT-VC per scenari in remoto e mdoc-CBOR per scenari in prossimità).

A tal fine, l’Ente deve compilare la sezione `casi_d_uso` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Per riferimenti e istruzioni di compilazione vedi [Appendice A](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-a--casi-duso).

### **Definire il Data Model**

Il Sistema IT-Wallet consente all’utente di ottenere in formato digitale i propri documenti, titoli e certificati sotto forma di EAA. Gli EAA sono rilasciati dal Fornitore di Attestati Elettronici di Attributi sulla base dei dati forniti dalla Fonte Autentica tramite l'e-service.
L’Ente deve quindi definire quali dati fornirà e in quale ordine, affinché l'EAA prodotto risulti adeguato all'utilizzo da parte dell’utente. 

> A tal fine, l’Ente deve compilare la sezione `e_service.response.data_model ` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) così da definire i dettagli sui dati che verranno messi a disposizione (es. tipologia, obbligatorietà, formato, lunghezza massima consentita, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vedi [Appendice B](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-b--data-model).

Reviewer 2: Some links appear to be broken. Could we add a GitHub Action to automatically check for broken links?

Author: we already have that but probably it not scans this brand new folder named handbook

going to add it accordingly, thx

Author: done in bf251f7b8228581706475082c210d66aaeb4b6ec

> Nella sezione [Template PDND Data Model](#Template-PDND-Data-Model) sono riportati i riferimenti dei template Data Model pubblicati su PDND relativi ad alcune tipologie di attestati emessi in IT-Wallet; l'Ente allinea il proprio data model (sezione `e_service.response.data_model`) a tali template per garantire che i dati forniti siano conformi alla struttura dell'Attestato che verrà emesso dal Fornitore di Attestati di Attributi.

Reviewer: ```suggestion
Nella sezione [Template e-service PDND](#Template-eservice-PDND) sono riportati i template e-service pubblicati su PDND relativi ad alcune tipologie di Attestati IT-Wallet; l'Ente interessato può adottare uno dei template messi a disposizione oppure utilizzarlo come punto di partenza per la definizione del proprio data model (sezione `e_service.response.data_model`).
```
template name fixed

In conclusione, un'adeguata definizione del Data Model pone le basi per una corretta implementazione dell'e-service da pubblicare su PDND (vedi [Step 2](#step-2--pubblicazione-in-collaudo)) ma è altresì importante considerare e rispettare i seguenti requisiti tecnici: 

> - **Identificativo utente**: Qualora fosse necessario identificare l'utente, il Codice Fiscale (CF) rappresenta l'identificativo univoco da utilizzare per le chiamate all'e-service; 

Reviewer 2: Does this mean that `idANPR` can be removed from the AS YAML file?

Reviewer: No, I think this means that the CF must always be considered as a way to identify the user

Reviewer 2: Ok, since the AS yaml states that user can be identified by idANPR or CF, can we maybe include the idANPR in the sentence?

Author: @costanzarosati @LCestoni may we allow also ID ANPR or CF only?

Author: CF only is the answer

- **Completezza base di dati**: Ogni e-service pubblicato su PDND dovrà esporre un set di dati completo nel contenuto e negli attributi. È ammessa la pubblicazione di basi dati parziali relative a periodi temporali limitati.

### **Definire le casistiche di errore**

L'e-service messo a disposizione dall'Ente deve prevedere e gestire specifiche situazioni di errore che possono verificarsi nella fase di recupero dei dati da parte del Fornitore di Attestati di Attributi.

> A tal fine, l'Ente deve compilare la sezione `e_service.response.mappatura_errori `del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vedi [Appendice C](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-c--mappatura-errori).

Reviewer 2: Is it necessary to compile this, or can we simply reference the AS YAML file for errors?

Reviewer: The JSON file also contains information about user actions and instructions for resolving the error status.

Author: @gaecapdtd ^

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `e_service.response.mappatura_errori `del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire, consentendo comunque l'aggiunta di eventuali errori specifici, per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vedi [Appendice C](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-c--mappatura-errori).
```

> 

Reviewer: ```suggestion
Nella sezione [Template e-service PDND](#Template-eservice-PDND) sono riportati i template e-service pubblicati su PDND relativi ad alcune tipologie di Attestati IT-Wallet; l'Ente interessato può adottare uno dei template messi a disposizione oppure utilizzarlo come punto di partenza per la mappatura degli errori  (sezione `e_service.response.mappatura_errori`).
```
We propose to add this paragraph because the e-service template includes also the error details

Gli stati ammissibili per un Attestato Elettronico di Attributi sono i seguenti: 

- **Valido**: EAA emesso, nessun segnale di modifica su uno dei suoi attributi o sul suo stato, entro la data di scadenza. L'utente può utilizzarlo in ogni scenario d'uso;
- **Sospeso**: EAA temporaneamente non valido, in uno stato di reversibilità. L'utente deve aspettare che lo stato torni ad essere valido (es. Patente ritirata);
- **Non Valido**: EAA non più valido, in uno stato di irreversibilità. L'utente può solo eliminarlo e/o richiedere l'emissione di nuovo EAA che sovrascriva il precedente (es. Patente annullata).

> Oltre agli stati sopra elencati, è bene specificare che lo stato di un Attestato Elettronico può essere influenzato anche dalla **scadenza amministrativa** e/o dalla **scadenza tecnica**. Rispettivamente l'EAA può, quindi, assumere anche i seguenti stati: 

Reviewer 2: Perhaps we could clarify that the states listed above are required to populate the `state` field in the e-service, while the states listed below are not. Is that correct?

Reviewer: Only "da aggiornare" state is not to be considered when populating the state field in the "mappatura stati" section of the JSON file

Reviewer 2: I meant the array `stati` in the json can have the following values: `Valido`, `Non Valido`, `Sospeso` and `Scaduto`. The AS yaml provide only the following states: `VALID`, `INVALID` and `SUSPENDED`. Is this ok?

Author: done in e231f5423639f305b4306db9e9acb5d90a1056f0

- **Scaduto**: EAA con data di scadenza amministrativa superata. La scadenza amministrativa può essere definita dal Titolare di Fonte Autentica e, se ritenuta utile o necessaria, deve essere inclusa come attributo all'interno del data model dell'EAA per garantire messaggi informativi all'interno della soluzione IT-Wallet (es. La tua Patente scade tra 30 giorni);
- **Da aggiornare**: EAA con data di scadenza tecnica superata. La scadenza tecnica è definita del Fornitore di Attestati Elettronici di Attributi ed è impostata generalmente a 1 anno o comunque a un valore inferiore o uguale alla data di scadenza amministrativa. Tale scadenza ha l'obiettivo di richiedere un'azione di riemissione esplicita all'utente e mitigare rischi di sicurezza.

Per approfondimenti vai alle Specifiche Tecniche, sezione [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html). 

> A tal fine, l'Ente deve compilare la sezione `mappatura-stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) per definire i dettagli sugli stati relativi al ciclo di vita dell'EAA, ovvero dichiarare la condizione di applicabilità dei tre stati sopracitati e l'eventuale relativo messaggio informativo da esporre all'utente.

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `mappatura-stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) per definire i dettagli sugli stati relativi al ciclo di vita dell'EAA, ovvero dichiarare la condizione di applicabilità dei tre stati sopracitati e l'eventuale relativo messaggio informativo da esporre all'utente.

Nella sezione [Template e-service PDND](#Template-eservice-PDND) sono riportati i template e-service pubblicati su PDND relativi ad alcune tipologie di Attestati IT-Wallet; l'Ente interessato può adottare uno dei template messi a disposizione oppure utilizzarlo come punto di partenza per la mappatura degli stati  (sezione `e_service.response.mappatura_stati`).
```

We propose to add this paragraph because the e-service template includes also the error details

- [Template e-service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd-template.html)
In questo step, l'Ente interessato deve: 
### **Aderire alla Piattaforma Digitale Nazionale Dati (PDND)**

Per effettuare l'onboarding alla PDND, qualora l'Ente ancora non abbia aderito, consultare il [Manuale Operativo PDND Interoperabilità](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/guida-alladesione). 

### **Pubblicare l'e-service su PDND in collaudo**

L'Ente deve sviluppare e rilasciare in collaudo un e-service coerente con il Data Model precedentemente definito, in linea con le informazioni presenti nel [Manuale Operativo PDND Interoperabilità](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/guida-alladesione) e con le Specifiche Tecniche italiane. Per i dettagli operativi e le Specifiche Tecniche, consultare:

- [sezione e-service del Manuale Operativo PDND Interoperabilità](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/e-service)
- [sezione Titolare di Fonte Autentica delle Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)

> Contestualmente al flusso di pubblicazione dell'e-service, l'Ente deve allegare il file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) precedentemente compilato in tutte le sue parti e e validato (sezioni `casi_d_uso`, `data_model`, `mappatura_errori`, `mappatura_stati` e `assistenza`). 

Reviewer 2: Should the JSON be attached to the email sent to the issuer, or included in the PDND e-service documentation?

Reviewer 3: @andreamisuraca included in the PDND e-service documentation

Reviewer: ```suggestion
Contestualmente al flusso di pubblicazione dell'e-service, l'Ente deve allegare, come documentazione aggiuntiva su PDND, il file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) precedentemente compilato in tutte le sue parti e e validato (sezioni `casi_d_uso`, `data_model`, `mappatura_errori`, `mappatura_stati` e `assistenza`). 
```
To better clarify the expected task

> Nel caso di EAA di interesse pubblico, l'Ente deve abilitare IPZS alla fruizione dell'e-service, se possibile con abilitazione automatica.

Reviewer 2: I would rephrase in:

"Nel caso di EAA di interesse pubblico, l'Ente deve aggiungere l'attributo certificato `Istituto Poligrafico e Zecca dello Stato S.P.A.` all'e-service erogato, se possibile impostando l'accetazione automatica delle richieste di fruizione".

> Nel caso di EAA di interesse pubblico, l'Ente deve abilitare IPZS alla fruizione dell'e-service, se possibile con abilitazione automatica.

Author: ```suggestion
Nel caso di EAA di interesse pubblico, l'Ente deve aggiungere l'attributo certificato Istituto Poligrafico e Zecca dello Stato S.P.A. all'e-service erogato, se possibile impostando l'accettazione automatica delle richieste di fruizione.
```

next time do a suggestion directly, it would be fair :-)

> Si consiglia di nominare l'e-service in "Creazione EAA [Nome / Nome tipologia EAA] – IT-Wallet" (es. "Creazione EAA Patente di guida – IT-Wallet" oppure "Creazione EAA Titoli di studio – IT-Wallet") e di predisporre una descrizione in linea con il [Manuale Operativo PDND Interoperabilità](https://italia.github.io/pdnd-guida-nomenclatura-eservice/index.html) referenziata nella Guida all'adesione PDND. 

Reviewer 2: That link does not point to the PDND manual. It would be better to reference the AgID PDND guidelines (Attachment 7), as this is the official document explaining how to name an e-service.

Reviewer: @andreamisuraca is this the right link? https://www.agid.gov.it/sites/agid/files/2025-06/Linee_guida_PDND_v2_allegato_7.pdf

Author: may we ask you the official href to that?

I'd suggest a direc github suggestion to speed up things

Reviewer 2: ```suggestion
Si consiglia di nominare l'e-service in "Creazione EAA [Nome / Nome tipologia EAA] – IT-Wallet" (es. "Creazione EAA Patente di guida – IT-Wallet" oppure "Creazione EAA Titoli di studio – IT-Wallet") e di predisporre una descrizione in linea con le Linee Guida sull’infrastruttura tecnologica della PDND per l’interoperabilità dei sistemi informativi e delle basi di dati ([Allegato 7 - Regole di popolamento](https://italia.github.io/pdnd-guida-nomenclatura-eservice/index.html)) emanate da AgID. 
```

### **Valutare attività di comunicazione**

L'Ente può prevedere attività di comunicazione finalizzate a informare gli utenti della possibilità di ottenere e utilizzare l'EAA prodotto con i propri dati all'interno di IT-Wallet. Per approfondimento, vai alle Specifiche Tecniche, sezione [Brand Identity](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/brand-identity.html) del Sistema IT-Wallet. 

## Step 7 | Manutenzione e assistenza

Una volta resi disponibili agli utenti gli EAA prodotti con i propri dati, la Fonte Autentica deve mantenere l'e-service e un ruolo attivo sia nella gestione dei dati (e del conseguente ciclo di vita degli EAA) che nell'assistenza agli utenti. 

Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:

- [Assistenza utente](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente)
In questo step, l'Ente interessato deve:
### **Garantire la gestione e manutenzione dell'e-service**

> L'Ente deve garantire il corretto funzionamento dell'e-service nel tempo, programmare adeguate azioni di monitoraggio e aggiornamento se richieste, ad esempio, da cambiamenti normativi o procedurali (es. nuovi dati, stati, casistiche di errore, etc.). 

Reviewer 2: I would also mention that the AS MUST fix any bugs by creating a new version of the e-service, and that it MUST create new versions to stay aligned with the latest patch version of the yaml published in the LTS branch of the guidelines. Do you agree?

Author: yes, please proceed with a suggestion

Reviewer: @andreamisuraca feel free to propose a suggested change to integrate your feedback

Reviewer 2: ```suggestion
L'Ente deve garantire il corretto funzionamento dell'e-service nel tempo, programmare adeguate azioni di monitoraggio e aggiornamento se richieste, ad esempio, da cambiamenti normativi o procedurali (es. nuovi dati, stati, casistiche di errore, etc.).

È altresì importante che l’ente mantenga il proprio e-service aggiornato all’ultima versione indicata dalle Linee Guida IT-Wallet, facendo riferimento al branch LTS e assicurandosi in particolare di implementare sempre l’ultima versione di patch disponibile.
```

- **Contribuire alla risoluzione di bug** 
Il referente dell'ambito sistemistico e il referente dell'ambito applicativo, così come definito nella sezione `assistenza.referenti` del template, devono contribuire alla diagnosi congiunta delle segnalazioni ricevute da Fornitore di Attestati Elettronici di Attributi (IPZS, nel caso di EAA di interesse pubblico) e Fornitori di Wallet (PagoPA, nel caso della soluzione pubblica IT-Wallet) e relativa risoluzione, secondo quanto definito dal [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet. 
- **Garantire il supporto agli utenti** 
Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione `assistenza` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json), devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.

## File da compilare

> 

Reviewer: ```suggestion
Per assolvere a quanto previsto dallo Step 1, l‘Ente deve: 
```

Reviewer: Typo: missing sentence

- scaricare il file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json)
- duplicare il file JSON per ciascuna EAA di interesse e rinomina ciascun file in "progettazione_ caratteristiche_eaa_[Nome Ente Titolare]_[Nome EAA]" (es. "progettazione_eaa_mim_titolo_di_studio.json"). 

> - compilare ciascun file Json in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di dati esposti, casi d'uso, assistenza, mappatura errori e stati.  

Reviewer: ```suggestion
- compilare ciascun file JSON in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di dati esposti, casi d'uso, assistenza, mappatura errori e stati.  
```

> - validare ciascun il file Json compilato utilizzando lo strumento indicato nel [file di validazione](https://github.com/italia/eid-wallet-it-docs/pull/1063/changes#diff-2337c8d8533155cad677214d3d5b723d77fa1f18fe26b99df6e9f6c91b4da9f5).

Reviewer: ```suggestion
- validare ciascun il file JSON compilato utilizzando lo strumento indicato nel [file di validazione](https://github.com/italia/eid-wallet-it-docs/pull/1063/changes#diff-2337c8d8533155cad677214d3d5b723d77fa1f18fe26b99df6e9f6c91b4da9f5).
```

Il file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) contiene le seguenti sezioni: 

- Casi d’uso (istruzioni di compilazione in [Appendice A](#appendix-a)) 
- Data Model (istruzioni di compilazione in [Appendice B](#appendix-b)) 
- Mappatura errori (istruzioni di compilazione in [Appendice C](#appendix-c)) 
- Mappatura stati (istruzioni di compilazione in [Appendice D](#appendix-d)) 
- Assistenza (istruzioni di compilazione in [Appendice E](#appendix-e)) 

## Appendice A – Casi d'uso EAA

Questa appendice descrive le istruzioni di compilazione della sezione `casi_d_uso` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire.

### Obiettivo

L’obiettivo della sezione `casi_d_uso` è quello di supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti. 

### Istruzioni di compilazione 

> 1. **metadata**: inserisci `nome_ente_titolare`, `nome_eaa` e `data_compilazione` (formato ISO: `AAAA-MM-GG`)

Reviewer: ```suggestion
1. **metadata**: inserisci `nome_ente_titolare`, `nome_eaa` e `data_compilazione` (formato ISO: `AAAA-MM-GG`);
```

> 2. **campi opzionali** e **campi obbligatori**: nel caso l'EAA si riferisce a un documento fisico o digitale già esistente (es. patente) compila sia i campi obbligatori che i campi opzionali (`volume_rilascio`, `canali_richiesta`, `formato_attuale,` `pagamento`, ecc.). Nel caso invece l'EAA non abbia un documento preesistente corrispondente, compila solo i campi obbligatori e lascia vuoti i campi opzionali. 

Reviewer: ```suggestion
2. **campi opzionali** e **campi obbligatori**: nel caso l'EAA si riferisce a un documento fisico o digitale già esistente (es. patente) compila sia i campi obbligatori che i campi opzionali (`volume_rilascio`, `canali_richiesta`, `formato_attuale,` `pagamento`, ecc.). Nel caso invece l'EAA non abbia un documento preesistente corrispondente, compila solo i campi obbligatori e lascia vuoti i campi opzionali;
```

> 3. **risposta**: compila il campo `risposta` rispondendo in maniera chiara ed esaustiva ad ogni domanda; il campo `suggerimento` è solo indicativo

Reviewer: ```suggestion
3. **risposta**: compila il campo `risposta` rispondendo in maniera chiara ed esaustiva ad ogni domanda; il campo `suggerimento` è solo indicativo.
```

        "risposta": "Tutti i cittadini italiani in possesso di diploma rilasciato da istituto italiano"
      }
    }
  }
}
```

---
## Appendice B – Data Model

Questa appendice descrive le istruzioni di compilazione della sezione `data_model` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

**Obiettivo**

L’obiettivo della sezione `data_model `è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di dati resi disponibili al suo interno. 

**Istruzioni di compilazione**

1. Prima di iniziare la compilazione, consulta i [Template PDND Data Model](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md#Template-PDND-Data-Model) e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche. 

> 2. Associa a ciascun dato che si intende rende disponibile all’interno dell’EAA un "nome campo" tra quelli definiti nella Lista nome campo sottostante o, se necessario, creane uno nuovo assicurandoti che sia parlante e che descriva adeguatamente il dato. 

Reviewer: ```suggestion
2. Associa a ciascun dato che si intende rende disponibile all’interno dell’EAA un "nome campo" tra quelli definiti nella "Lista nome campo" sottostante o, se necessario, creane uno nuovo assicurandoti che sia parlante e che descriva adeguatamente il dato. 
```

```json
"e_service": {
  "lista_nome_campo": [
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Nome",
      "descrizione": ""
    },
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Cognome",
      "descrizione": ""
    },
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Codice Fiscale",
      "descrizione": ""
    },
    {
      "categoria": "Residenza e domicilio",
      "parametro": "",
      "nome_campo": "Comune di residenza",
      "descrizione": ""
    },
    {
      "categoria": "Istruzione e formazione",
      "parametro": "",
      "nome_campo": "Qualifica",
      "descrizione": ""
    },
    {
      "categoria": "Patenti e veicoli",
      "parametro": "",
      "nome_campo": "Data di rilascio",
      "descrizione": ""
    },
    {
      "categoria": "Patenti e veicoli",
      "parametro": "",
      "nome_campo": "Scadenza",
      "descrizione": ""
    },
    {
      "categoria": "Generali",
      "parametro": "",
      "nome_campo": "Origine dei dati",
      "descrizione": ""
    }
  ]
}
```

> *Tabella 2 – Lista nome campo (sezione `e_service.lista_nome_campo`)*

Reviewer: @peppelinux should be the "Lista nome campo" coherent with the JSON file? The list doesn't seem complete right now. If the aim is to represent just an example, we suggest to rename the table "Tabella 2 . Estratto da lista nome campo"

Author: it is coherent
https://github.com/italia/eid-wallet-it-docs/pull/1063/changes#diff-cbd5459b99cbcecdb624eeb019d2c5eb83b1898bdf0c6e1e6f4ae30911c0d97eR467

Reviewer: The comment can be closed (I tried, but I got an error message)

> ## Appendice C – Mappatura errori

Reviewer 2: If this section is necessary, it should be aligned with the YAML published in the guidelines.

Author: @ChiaLeon @RosaliaGaleano do you agree?

Reviewer: Yes, we agree. Follow below the suggested changes

Questa appendice descrive le istruzioni di compilazione della sezione `mappatura_errori` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

**Obiettivo**

L’obiettivo della sezione `mappatura_errori` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di errori che potrebbero occorrere interagendo con l'e-service corrispondente.

**Istruzioni di compilazione**

> 1. Definisci la motivazione che ha scatenato l'errore e popola il campo "Causa" per ciascuna casistica (es. Servizio momentaneamente non disponibile). 

Reviewer: ```suggestion
1. Per il codice 200 e per tutti gli errori obbligatori (400, 401, 404, 429, 500, 503) definisci la motivazione che ha scatenato l'errore e popola il campo "Causa"(es. Servizio momentaneamente non disponibile);
2. Per ciascun errore descrivi l'azione necessaria per risolvere il problema nel campo "Azione utente". (es. Ti invitiamo a riprovare più tardi). Usa il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica;
3. Se ritenuto utile, compila allo stesso modo gli errori non obbligatori (es. 540 e 541) e/o aggiungi eventuali ulteriori errori specifici.

Nel caso di compilazione degli errori opzionali:
```
Suggested changes to guarantee alignment with technical specifications in terms of errors (see [PDND-AS-yaml](https://github.com/italia/eid-wallet-it-docs/blob/versione-corrente/docs/en/oas3/OAS3-PDND-AS.yaml))
cc @andreamisuraca

> 2. Per l'errore 540 (EAA non esistente presso l'Authentic Source), utilizza il formato "state": "description", es.: "NOT_EXISTING": "l'EAA non è presente presso l'Authentic Source", "PENDING": "l'EAA è in attesa di emissione". 

Reviewer: ```suggestion
- Per l'errore 540 (EAA non esistente presso l'Authentic Source), utilizza il formato "state": "description", es.: "NOT_EXISTING": "l'EAA non è presente presso l'Authentic Source", "PENDING": "l'EAA è in attesa di emissione". 
```

> 3. Per l'errore 541 (EAA in stato non valido o sospeso), descrivi la causa secondo le [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/OAS3-PDND-Issuer.html#tag/e-services-PDND/operation/notifyStatusCredentials) (es. scaduto, sospeso, revocato, annullato). 

Reviewer: ```suggestion
- Per l'errore 541 (EAA in stato non valido o sospeso), descrivi la causa secondo le [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/OAS3-PDND-Issuer.html#tag/e-services-PDND/operation/notifyStatusCredentials) (es. scaduto, sospeso, revocato, annullato). 
```

> 4. Descrivi l'azione necessaria per risolvere il problema nel campo "Azione utente". (es. Ti invitiamo a riprovare più tardi). Usa il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica. 

Reviewer: ```suggestion

```
Text moved upward (see previous comment)

> 5. Aggiungi nuovi codici errore se necessari per lo specifico attestato.

Reviewer: ```suggestion

```
Text moved upward (see previous comment)

```json
"e_service": {
  "response": {
    "mappatura_errori": [
      {
        "codice": "200",
        "esito": "Attestato digitale valido",
        "applicabile": "SI",
        "causa": "Vengono ritornati correttamente i dati, nessuna risposta di errore. Qualora non ci fossero azioni utente da eseguire, riportare stringa vuota.",
        "azione_utente": "",
        "note": ""
      },
      {
        "codice": "404",
        "esito": "Not found",
        "applicabile": "SI",
        "causa": "Non sono stati trovati documenti di titolarità dell'utente",

>         "azione_utente": "Chiudere e riprovare successivamente",

Reviewer: ```suggestion
        "azione_utente": "Chiudere e riprovare successivamente",
        "messaggio": "Ti invitiamo ad ottenere il documento presso l'Ente titolare prima di richiedere la sua versione digitale"
```

We suggest to add the "messaggio" field to be coherent with the "appendix D mappatura stati".
@peppelinux if you agree, you should add this "messaggio" field here in the example and inside the JSON files (template / validator)

        "note": "L'utente deve prima acquisire la titolarità del documento per ottenerne la versione digitale"
      },
      {
        "codice": "540",
        "esito": "EAA non esistente presso l'Authentic Source",
        "applicabile": "",
        "causa": "",
        "azione_utente": "",
        "note": ""
      },
      {
        "codice": "541",
        "esito": "EAA in stato non valido o sospeso",
        "applicabile": "",
        "causa": "",
        "azione_utente": "",
        "note": ""
      }
    ]
  }
}
```

*Tabella 3 – Mappatura Errori (sezione `e_service.response.mappatura_errori`)*

> ## Appendice C – Mappatura stati

Reviewer: ```suggestion
## Appendice D – Mappatura stati
```
typo

```json
"e_service": {
  "response": {
    "stati": [
      {
        "stato": "Valido",
        "descrizione": "L'EAA è valido e può essere utilizzato",
        "applicabile": "SI",
        "azione_utente": "",
        "messaggio": "",
        "note": ""
      },
      {
        "stato": "Non Valido",
        "descrizione": "L'EAA non è più valido e dunque non può essere più utilizzato",
        "applicabile": "",
        "azione_utente": "L'utente deve scaricare nuovamente l'EAA in app",
        "messaggio": "",
        "note": ""
      },
      {
        "stato": "Sospeso",
        "descrizione": "L'EAA è sospeso e non può essere temporaneamente utilizzato",
        "applicabile": "",
        "azione_utente": "",
        "messaggio": "",
        "note": ""
      },
      {
        "stato": "Scaduto",
        "descrizione": "L'EAA è scaduto e necessita una riemissione",
        "applicabile": "",
        "azione_utente": "",
        "messaggio": "",
        "note": ""
      }
    ]
  }
}
```

*Tabella 4 – Mappatura stati (sezione `e_service.response.stati`)*

## Appendice E – Assistenza

> Questa appendice descrive le istruzioni di compilazione della sezione `assistenza` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

Reviewer 3: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `assistenza` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

**Obiettivo** 

L’obiettivo della sezione `assistenza` è quello di supportare gli Enti nella definizione dei contenuti per l'informazione e il supporto all'utente nell’interazione con l'EAA.  

Nello specifico, l'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili i recapiti necessari per la risoluzione di eventuali malfunzionamenti, i canali disponibili per la gestione dell'assistenza agli utenti e i contenuti utili alla predisposizione di nuove Domande Frequenti e/o testi informativi da esporre all’utente in IT-Wallet. 

**Istruzioni di compilazione** 

- Referenti: fornisci e mantieni aggiornati, i dati di contatto (nome, cognome, email, telefono) di almeno 1 referente per l’assistenza agli utenti, 1 referente in ambito applicativo e 1 referente in ambito sistemistico che possano prontamente collaborare con il Fornitore di Attestati Elettronici di Attributi (IPZS per gli Attestati Elettronici di interesse pubblico) e con il Fornitore di Wallet (PagoPA per la soluzione IT-Wallet pubblica) per la risoluzione congiunta di segnalazioni degli utenti e/o malfunzionamenti tra i servizi. 
- Canali: fornisci almeno un canale di assistenza (es. e-mail assistenza, numero di telefono, etc.) di responsabilità dell’Ente che rappresenti, nel ruolo di Titolare di Fonte Autentica, a cui si possa indirizzare l’utente per quelle richieste di supporto e segnalazioni non gestibili all’interno della soluzione IT-Wallet.  
- FAQ: contribuisci alla definizione dei contenuti utili a predisporre le domande più frequenti e comuni relative all’EAA in analisi, dall’emissione all’utilizzo fino alla gestione del suo ciclo vita. 
- testi_informativi (opzionale): se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 
1) Poniti le seguenti domande (puoi fare riferimento alla sezione “casi_d_uso”, già compilato):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
2) Formula un testo informativo rivolto all’utente a partire dai contenuti sopra raccolti, assicurandoti che sia chiaro, semplice, diretto e conciso (circa 300 - 450 caratteri, spazi inclusi).
```

typo: missing text related to Assistance section lost in the reivew process

> ## Appendice B – Data Model

Reviewer 2: Why do we start again with B after E?

Author: it seems a typo

@ChiaLeon ^

Reviewer: I think this is a typo, everything from here to "Template PDND Data Model" is a repetition and needs to be deleted

Author: @ChiaLeon ^

Reviewer 3: @peppelinux Done. Proposed deletion as suggested change below

```json
"e_service": {
  "lista_nome_campo": [
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Nome",
      "descrizione": ""
    },
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Cognome",
      "descrizione": ""
    },
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Codice Fiscale",
      "descrizione": ""
    },
    {
      "categoria": "Residenza e domicilio",
      "parametro": "",
      "nome_campo": "Comune di residenza",
      "descrizione": ""
    },
    {
      "categoria": "Istruzione e formazione",
      "parametro": "",
      "nome_campo": "Qualifica",
      "descrizione": ""
    },
    {
      "categoria": "Patenti e veicoli",
      "parametro": "",
      "nome_campo": "Data di rilascio",
      "descrizione": ""
    },
    {
      "categoria": "Patenti e veicoli",
      "parametro": "",
      "nome_campo": "Scadenza",
      "descrizione": ""
    },
    {
      "categoria": "Generali",
      "parametro": "",
      "nome_campo": "Origine dei dati",
      "descrizione": ""
    }
  ]
}
```

---

> ## Template PDND Data Model

Reviewer: ```suggestion
## Template e-service PDND
```

Reviewer: Since e-service PDND includes data model, states and errors, we suggested to refer to this chapter in the step 1 relative sections (see comments upward)

> Di seguito si condividono i data model già definiti o contenuti nella soluzione pubblica IT-Wallet, che l'Ente dovrebbe prendere a riferimento per allineare il proprio dataset (sezione **e_service**) quando intende rendere disponibili dati appartenenti alle stesse categorie semantico-funzionali. In particolare: 

Reviewer: ```suggestion
Di seguito si condividono i template e-service PDND già definiti o contenuti nella soluzione pubblica IT-Wallet, che l'Ente dovrebbe prendere a riferimento per allineare il proprio e-service (sezione **e_service**) quando intende rendere disponibili dati appartenenti alle stesse categorie semantico-funzionali. In particolare: 
```

```json
"e_service": {
  "lista_nome_campo": [
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Nome",
      "descrizione": ""
    },
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Cognome",
      "descrizione": ""
    },
    {
      "categoria": "Dati anagrafici e di identità",
      "parametro": "",
      "nome_campo": "Codice Fiscale",
      "descrizione": ""
    },
    {
      "categoria": "Residenza e domicilio",
      "parametro": "",
      "nome_campo": "Comune di residenza",
      "descrizione": ""
    },
    {
      "categoria": "Istruzione e formazione",
      "parametro": "",
      "nome_campo": "Qualifica",
      "descrizione": ""
    },
    {
      "categoria": "Patenti e veicoli",
      "parametro": "",
      "nome_campo": "Data di rilascio",
      "descrizione": ""
    },
    {
      "categoria": "Patenti e veicoli",
      "parametro": "",
      "nome_campo": "Scadenza",
      "descrizione": ""
    },
    {
      "categoria": "Generali",
      "parametro": "",
      "nome_campo": "Origine dei dati",
      "descrizione": ""
    }
  ]
}
```

> 

Reviewer 3: ```suggestion

```

Typo, redoundant

| Patenti e veicoli             | Data di rilascio, Paese di rilascio, Categoria, Scadenza, Numero, Codici                                                                                                                                                                                                                                                                                                           |
| Generali                      | Origine dei dati, Emissione versione digitale                                                                                                                                                                                                                                                                                                                                      |

---

## Template PDND Data Model

Di seguito si condividono i data model già definiti o contenuti nella soluzione pubblica IT-Wallet, che l'Ente dovrebbe prendere a riferimento per allineare il proprio dataset (sezione **e_service**) quando intende rendere disponibili dati appartenenti alle stesse categorie semantico-funzionali. In particolare: 

> - EAA di affiliazione ad un ente o organizzazione 

Reviewer 3: link to add

Author: @gaecapdtd ^

Reviewer 4: @andreamisuraca come possiamo indicare dove consultare i data model?

Reviewer 2: As mentioned to @ChiaLeon and @RosaliaGaleano, once the templated as published in PDND we can add the links here.

Per la piena attuazione del Sistema IT-Wallet, all'articolo istitutivo del CAD seguono due decreti attuativi, di cui uno di adozione delle Linee Guida proposte da AgID, che vengono completate dalle Specifiche tecniche. La versione di riferimento delle Specifiche è quella mantenuta nel branch **Long-Term Support (LTS)** come descritto nel relativo [README](https://github.com/italia/eid-wallet-it-docs) all'interno della sezione [Branching Approach](https://github.com/italia/eid-wallet-it-docs?tab=readme-ov-file#branching-approach). La versione corrente delle Specifiche è disponibile a [questo link](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/) ([versione inglese](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/)).  

## Scopo e ambito di applicazione

Questo manuale, che sovrascrive e supera i manuali in formato .pdf precedentemente condivisi, ha lo scopo di: 

> 

Reviewer 3: ```suggestion

- supportare gli Enti nella definizione di tutti gli aspetti che contribuiscono a rendere possibile l’emissione, la fruizione e la manutenzione dell’EAA da parte dei soggetti interessati; 
```

  - [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)
  - [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)
  [Vai allo Step 3](#step-3--test-in-collaudo)
- **Step 4 | Pubblicazione in produzione**: l'Ente rilascia l'e-service in ambiente di produzione su PDND e attiva il relativo servizio Signal Hub in produzione. Infine, l'Ente notifica al Fornitore di Attestati Elettronici di Attributi. Per i dettagli implementativi, consultare le Specifiche Tecniche italiane:
  - [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)
  - [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)
  [Vai allo Step 4](#step-4--pubblicazione-in-produzione)

> - **Step 5 | Test in produzione**: l'Ente, in ambiente di produzione, esegue i test di integrazione, di carico e long run dell'e-service. Per i dettagli implementativi, consultare le Specifiche Tecniche italiane:

Reviewer 3: ```suggestion
- **Step 5 | Test in produzione**: l'Ente, in ambiente di produzione, esegue i test di integrazione, di carico e long run dell'e-service con il Fornitore di Attestati Elettronici di Attributi e, ove possibile se richiesto, con il Fornitore di Wallet per testare anche gli aspetti relativi alla UX/UI.
```

L'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili contenuti utili alla predisposizione di nuove Domande Frequenti e/o testi informativi in app, e fornendo i recapiti necessari per la gestione dell'assistenza agli utenti.  
A tal fine, l'Ente deve compilare la sezione **assistenza** nel [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json) (vedi [Appendice B](#appendice-b--assistenza-eaa)). 

### **Definire le caratteristiche grafiche dell'EAA prodotto dai propri dati**

La rappresentazione grafica dell'EAA nell'IT-Wallet può dipendere, per specifici aspetti, da parametri definiti nelle Specifiche Tecniche, sezione [Focus sugli Attestati Elettronici di Attributi](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#focus-sugli-attestati-elettronici-di-attributi). 
L'Ente interessato a personalizzare la resa grafica dell'EAA prodotto dai propri dati deve trasmettere i propri materiali come documentazione allegata all'e-service su PDND.  

> 

Reviewer 3: ```suggestion
### **Validare il file “progettazione_EAA”**

A conclusione delle azioni sopra elencate, l’Ente deve validare il file ”progettazione_EAA" in tutte le sue parti (sezioni `casi_d_uso`, `data_model`, `mappatura_errori`, `mappatura_stati` e `assistenza`) utilizzando lo strumento indicato nel file ”validazione_progettazione_caratteristiche_EAA”. 

Per poter proseguire con gli step successivi, è infatti obbligatorio eseguire validazione JSON Schema e controllo sintattico. Per i comandi e il workflow, vedi [Validazione JSON Schema e Linter](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/validazione-json-schema-linter.md). La checklist prevede: 

- validazione JSON Schema superata; 
- JSON Linter senza errori; 
- tutti i campi obbligatori (risposta) compilati per la sezione scelta; 
- file rinominato correttamente.
```

  - Nel caso non esista ad oggi documento, compila solo i campi obbligatori e lascia vuoti quelli opzionali relativi al documento esistente
2. Compila allo stesso modo i file delle altre EAA di interesse.

### Validazione e checklist pre-sottomissione

**Prima di inviare il file**, è obbligatorio eseguire validazione JSON Schema e controllo sintattico. Per i comandi e il workflow, vedi [Validazione JSON Schema e Linter](validazione-json-schema-linter.md).

**Checklist**:

- Validazione JSON Schema superata
- JSON Linter senza errori
- Tutti i campi obbligatori (`risposta`) compilati
- File rinominato correttamente e pronto per lo Step 2

### Dopo la compilazione

1. Condivi i file adeguatamente rinominati e compilati, secondo le modalità previste dallo Step 2.
2. Prosegui con le fasi successive descritte nel manuale.
3. Mantieni sempre aggiornate le informazioni secondo le modalità definite nello Step 7.

### Sezione casi d'uso (documento esistente o non)

Aree tematiche: **Target utenti**, **Emissione e formato**, **Utilizzo**, **Pagamento** (solo se documento esistente), **Legale e privacy**. I campi relativi al documento esistente sono opzionali.

---

> 

Reviewer 3: ```suggestion
## Appendice B – Data Model

Questa appendice descrive le istruzioni di compilazione della sezione `data_model` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

**Obiettivo**

L’obiettivo della sezione `data_model `è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di dati resi disponibili al suo interno. 

**Istruzioni di compilazione**

1. Prima di iniziare la compilazione, consulta i [Template PDND Data Model](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#Template-PDND-Data-Model) e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche. 
2. Associa a ciascun dato che si intende rende disponibile all’interno dell’EAA un "nome campo" tra quelli definiti nella Lista nome campo sottostante o, se necessario, creane uno nuovo assicurandoti che sia parlante e che descriva adeguatamente il dato. 
3. Ordina i campi in modo da facilitare la leggibilità: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato.

| ATTESTAZIONE | PARAMETRO | DESCRIZIONE                | NOME CAMPO     | ESEMPIO CAMPO COMPILATO | OBBLIGATORIO | TIPOLOGIA    | LUNGHEZZA MASSIMA CARATTERI | NOTE |
| ------------ | --------- | -------------------------- | -------------- | ----------------------- | ------------ | ------------ | --------------------------- | ---- |
| ISEE         | tax_code  | codice fiscale dell'utente | Codice Fiscale | DLNRSL88L51C348G        | SI           | ALFANUMERICO | 16                          |      |

Tabella 1 Struttura Data Model 

| CATEGORIA                     | NOME CAMPO                                                                                                                                                                                                                                                                                                                                                                         |
| ----------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Dati anagrafici e di identità | Nome, Cognome, Data di nascita, Luogo di nascita, Codice Fiscale, Nazionalità                                                                                                                                                                                                                                                                                                      |
| Residenza e domicilio         | Comune di residenza, Frazione, Indirizzo, Seconda riga indirizzo, Codice postale, Provincia / Stato / Regione, Residente dal, Residenza estera                                                                                                                                                                                                                                     |
| Istruzione e formazione       | Diploma conclusivo del primo/secondo ciclo, Diploma, Descrizione diploma, Istituto, Codice istituto, Qualifica, Qualifica professionale, Plesso scolastico, Codice plesso, Tipologia di corso, Corso di laurea, Classe di laurea, Anno di corso, Anno accademico/scolastico, Data di conseguimento, Periodo didattico, Tipologia di frequenza, Data di scadenza, Voto, Voto finale |
| Sanità e previdenza           | Diritto accompagnatore                                                                                                                                                                                                                                                                                                                                                             |
| Patenti e veicoli             | Data di rilascio, Paese di rilascio, Categoria, Scadenza, Numero, Codici                                                                                                                                                                                                                                                                                                           |
| Generali                      | Origine dei dati, Emissione versione digitale                                                                                                                                                                                                                                                                                                                                      |

Tabella 2 Lista nome campo

## Appendice C – Mappatura errori

Questa appendice descrive le istruzioni di compilazione della sezione `mappatura_errori` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

**Obiettivo**

L’obiettivo della sezione `mappatura_errori` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di errori che potrebbero occorrere interagendo con l'e-service corrispondente.

**Istruzioni di compilazione**

1. Definisci la motivazione che ha scatenato l'errore e popola il campo "Causa" per ciascuna casistica (es. Servizio momentaneamente non disponibile). 
2. Per l'errore 540 (EAA non esistente presso l'Authentic Source), utilizza il formato "state": "description", es.: "NOT_EXISTING": "l'EAA non è presente presso l'Authentic Source", "PENDING": "l'EAA è in attesa di emissione". 
3. Per l'errore 541 (EAA in stato non valido o sospeso), descrivi la causa secondo le [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/OAS3-PDND-Issuer.html#tag/e-services-PDND/operation/notifyStatusCredentials) (es. scaduto, sospeso, revocato, annullato). 
4. Descrivi l'azione necessaria per risolvere il problema nel campo "Azione utente". (es. Ti invitiamo a riprovare più tardi). Usa il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica. 
5. Aggiungi nuovi codici errore se necessari per lo specifico attestato.

| CODICE  | ESITO                                                                | APPLICABILE | CAUSA                                          | AZIONE UTENTE                        | NOTE |
| ------- | -------------------------------------------------------------------- | ----------- | ---------------------------------------------- | ------------------------------------ | ---- |
| 200     | Attestato digitale valido                                            | SI          | Dati ritornati correttamente                   | N/A                                  |      |
| 400-451 | Errori client (Bad Request, Unauthorized, ecc.)                      |             |                                                |                                      |      |
| 404     | Not found                                                            | SI          | Non sono stati trovati documenti di titolarità | Chiudere e riprovare successivamente |      |
| 500-503 | Errori server (generico, servizio non implementato, non disponibile) |             |                                                |                                      |      |
| 540     | EAA non esistente presso l'Authentic Source                          |             | Formato `"state": "description"`               |                                      |      |

Tabella 3 Mappatura Errori

## Appendice C – Mappatura stati

Questa appendice descrive le istruzioni di compilazione della sezione `mappatura_stati` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

**Obiettivo**

L’obiettivo della sezione `mappatura_stati `è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.

**Istruzioni di compilazione**

1. Mappa la condizione di applicabilità di ciascuno stato relativamente all'attestato in analisi. 
2. Descrivi l'azione necessaria per ripristinare lo stato di validità all’interno del campo "Azione utente" (es. Chiedere la riemissione del documento presso uffici fisici / in digitale).  
3. Definisci il messaggio da condividere con l'utente (es. I tuoi dati sono stati aggiornati nella banca dati ANIS, scarica la nuova versione digitale del documento). Usa il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica. 
4. Per approfondimenti: [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html).

| STATO      | DESCRIZIONE                                    | APPLICABILE | AZIONE UTENTE                                         | MESSAGGIO | NOTE |
| ---------- | ---------------------------------------------- | ----------- | ----------------------------------------------------- | --------- | ---- |
| Valido     | L'attestato è valido e può essere utilizzato   | SI          | N/A                                                   |           |      |
| Non Valido | L'attestato non è più valido                   |             | L'utente deve scaricare nuovamente l'attestato in app |           |      |
| Sospeso    | L'attestato è temporaneamente non utilizzabile |             |                                                       |           |      |
| Scaduto    | L'attestato è scaduto e necessita riemissione  |             |                                                       |           |      |

Tabella 4 Mappatura stati

```

Data model moved upward (Appendix B)
Lista nome campo moved upward in Appendix B, since related to data model
Mappatura Errori moved upward (Appendix C)
Mappatura Stati moved upward (Appendix D)

- [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)
In questo step, l'Ente interessato deve:
### **Pubblicare l'e-service su PDND in produzione**

L'Ente deve rilasciare l'e-service su PDND in produzione. Nel caso di EAA di interesse pubblico, l'Ente deve abilitare IPZS alla fruizione del servizio, se possibile con abilitazione automatica. 

### **Attivare il servizio Signal Hub in produzione**

L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `e_service.response.stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json).  

### **Portare in produzione il Credential Offer (opzionale)**

L'Ente, se precedentemente sviluppato in collaudo, deve provvedere in questo step al rilascio in produzione del Credential Offer all'interno delle proprie soluzioni.  

### **Notificare l'avvenuto rilascio in produzione**

> L'Ente deve notificare il Fornitore di Attestati di Attributi circa l'avvenuto rilascio dell'e-service in produzione su PDND, dei servizi di Signal Hub in produzione e, se previsto, del Credential Offer. In caso di EAA di interesse pubblico, l'Ente deve notificare IPZS inviando una mail all'indirizzo identitadigitale@pec.ipzs.it e, in caso di EAA disponibile nella soluzione pubblica di IT-Wallet, l'Ente deve notificare PagoPA inviando una mail all'indirizzo XX.

Author: ```suggestion
L'Ente deve notificare il Fornitore di Attestati di Attributi circa l'avvenuto rilascio dell'e-service in produzione su PDND, dei servizi di Signal Hub in produzione e, se previsto, del Credential Offer. In caso di EAA di interesse pubblico, l'Ente deve notificare IPZS inviando una mail all'indirizzo e-mail [identitadigitale@pec.ipzs.it](mailto:identitadigitale@pec.ipzs.it) e, in caso di EAA disponibile nella soluzione pubblica di IT-Wallet, l'Ente deve notificare PagoPA inviando una mail all'indirizzo e-mail [itwallet@pagopa.it](mailto:itwallet@pagopa.it).
```

Author: @silviafulci may we have an organizational e-mail instead of a personal one?

Reviewer 5: Please use it: itwallet@pagopa.it

- [Scopo e ambito di applicazione](#Scopo-e-ambito-di-applicazione)
- [Ruoli e responsabilità del Titolare di Fonte Autentica](#Ruoli-e-responsabilità-del-Titolare-di-Fonte-Autentica)
- [Come diventare Titolare di Fonte Autentica](#Come-diventare-Titolare-di-Fonte-Autentica)
  - [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa) 
  - [Step 2 | Pubblicazione in collaudo](#step-2--pubblicazione-in-collaudo) 
  - [Step 3 | Test in collaudo](#step-3--test-in-collaudo)
  - [Step 4 | Pubblicazione in produzione](#step-4--pubblicazione-in-produzione) 
  - [Step 5 | Test in produzione](#step-5--test-in-produzione)
  - [Step 6 | Pianificazione rilascio EAA](#step-6--pianificazione-rilascio-eaa)
  - [Step 7 | Manutenzione e assistenza](#step-7--manutenzione-e-assistenza)
- [File da compilare](#file-da-compilare)
- [Appendice A – Casi d'uso](#appendice-a--casi-duso)
- [Appendice B – Data Model](#appendice-b--data-model)
- [Appendice C – Mappatura errori](#appendice-c--mappatura-errori)
- [Appendice D – Mappatura stati](#appendice-d--mappatura-stati)
- [Appendice E – Assistenza](#appendice-e--assistenza)

> - [Template e-service PDND](#Template-eservice-PDND)

Reviewer 3: ```suggestion

```

"Template e-service PDND" section deleted

Il presente manuale è da intendersi anche come uno strumento aperto alla condivisione e alla collaborazione tra gli Enti interessati. 

## Ruoli e responsabilità del Titolare di Fonte Autentica

Il Titolare di Fonte Autentica (Authentic Source - AS) è il soggetto che, nel Sistema IT-Wallet, detiene i dati da cui vengono creati gli Attestati Elettronici di Attributi rilasciati nelle soluzioni IT-Wallet. Ad esempio, per la Patente di Guida il Titolare di Fonte Autentica è la Direzione Generale di Motorizzazione del Ministero delle Infrastrutture e dei Trasporti. 

L'Ente che fornisce i dati, in quanto Titolare di Fonte Autentica, è l'unico soggetto che può definire mezzi e finalità di uso degli stessi. L'Ente rimarrà sempre il "proprietario" del dato e sarà responsabile del ciclo di vita del dato quindi di eventuali modifiche o cambiamenti di stato. Il Sistema IT-Wallet è concepito in modo che l'Ente possa continuare a gestire il dato in modo autonomo e conforme alle proprie politiche e alle normative vigenti. 

Di seguito rappresentato il ruolo del Titolare di Fonte Autentica e delle altre entità nel Sistema IT-Wallet. 

> ![Ruolo del Titolare di Fonte Autentica nel Sistema IT-Wallet](https://github.com/user-attachments/assets/e23b1cd0-b547-408e-8354-58131e0977b4)

Reviewer 3: @peppelinux to update image

Author: done

*Figura 1: Ruolo del Titolare di Fonte Autentica nel Sistema IT-Wallet*

Di seguito il ruolo del Titolare di Fonte Autentica nel contesto del flusso di richiesta ed emissione di un EAA. 

> ![Flusso di richiesta ed emissione di un EAA nel Sistema IT-Wallet](https://github.com/user-attachments/assets/9223e9fa-9830-4e1d-b1ac-6354554ef2ce)

Reviewer 3: @peppelinux to update image

Author: done

A tal fine, l’Ente deve compilare la sezione `casi_d_uso` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Per riferimenti e istruzioni di compilazione vedi [Appendice A](#appendice-a--casi-duso).

### **Definire il Data Model**

Il Sistema IT-Wallet consente all’utente di ottenere in formato digitale i propri documenti, titoli e certificati sotto forma di EAA. Gli EAA sono rilasciati dal Fornitore di Attestati Elettronici di Attributi sulla base dei dati forniti dalla Fonte Autentica tramite l'e-service.
L’Ente deve quindi definire quali dati fornirà e in quale ordine, affinché l'EAA prodotto risulti adeguato all'utilizzo da parte dell’utente. 

A tal fine, l’Ente deve compilare la sezione `e_service.response.data_model ` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) così da definire i dettagli sui dati che verranno messi a disposizione (es. tipologia, obbligatorietà, formato, lunghezza massima consentita, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vedi [Appendice B](#appendice-b--data-model).

> Nella sezione [Template e-service PDND](#Template-eservice-PDND) sono riportati i template e-service pubblicati su PDND relativi ad alcune tipologie di Attestati IT-Wallet; l'Ente interessato può adottare uno dei template messi a disposizione oppure utilizzarlo come punto di partenza per la definizione del proprio data model (sezione `e_service.response.data_model`).

Reviewer 3: ```suggestion

```

"Template e-service PDND" section deleted

- **Identificativo utente**: Qualora fosse necessario identificare l'utente, il Codice Fiscale (CF) rappresenta l'identificativo univoco da utilizzare per le chiamate all'e-service; 
- **Completezza base di dati**: Ogni e-service pubblicato su PDND dovrà esporre un set di dati completo nel contenuto e negli attributi. È ammessa la pubblicazione di basi dati parziali relative a periodi temporali limitati.

### **Definire le casistiche di errore**

L'e-service messo a disposizione dall'Ente deve prevedere e gestire specifiche situazioni di errore che possono verificarsi nella fase di recupero dei dati da parte del Fornitore di Attestati di Attributi.

A tal fine, l'Ente deve compilare la sezione `e_service.response.mappatura_errori `del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire, consentendo comunque l'aggiunta di eventuali errori specifici, per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vedi [Appendice C](#appendice-c--mappatura-errori).

> Nella sezione [Template e-service PDND](#Template-eservice-PDND) sono riportati i template e-service pubblicati su PDND relativi ad alcune tipologie di Attestati IT-Wallet; l'Ente interessato può adottare uno dei template messi a disposizione oppure utilizzarlo come punto di partenza per la mappatura degli errori  (sezione `e_service.response.mappatura_errori`).

Reviewer 3: ```suggestion

```

"Template e-service PDND" section deleted

Per approfondimenti vai alle Specifiche Tecniche, sezione [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html). 

A tal fine, l'Ente deve compilare la sezione `e_service.response.stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) per definire i messaggi e l'applicabilità dei quattro valori **Valido**, **Non Valido**, **Sospeso** e **Scaduto**.

> Nella sezione [Template e-service PDND](#Template-eservice-PDND) sono riportati i template e-service pubblicati su PDND relativi ad alcune tipologie di Attestati IT-Wallet; l'Ente interessato può adottare uno dei template messi a disposizione oppure utilizzarlo come punto di partenza per la mappatura degli stati (sezione `e_service.response.stati`).

Reviewer 3: ```suggestion

```

"Template e-service PDND" section deleted

- validazione JSON Schema superata; 
- JSON Linter senza errori; 
- tutti i campi obbligatori (risposta) compilati per la sezione scelta; 
- file rinominato correttamente.
## Step 2 | Pubblicazione in collaudo

Questo step ha l'obiettivo di rilasciare in collaudo su PDND l'e-service che espone i dati per la produzione degli EAA e di attivare il relativo servizio Signal Hub per la gestione dei dati nel tempo. Inoltre, l'Ente può rilasciare in collaudo il Credential Offer se ritenuto utile durate lo [Step 1](#step-1--progettazione-caratteristiche-eaa)).

Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:

- [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)
- [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)
- [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)

> - [Template e-service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd-template.html)

Reviewer 3: ```suggestion

```

"Template e-service PDND" section deleted

- [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)
In questo step, l'Ente interessato deve:
### **Pubblicare l'e-service su PDND in produzione**

L'Ente deve rilasciare l'e-service su PDND in produzione. Nel caso di EAA di interesse pubblico, l'Ente deve abilitare IPZS alla fruizione del servizio, se possibile con abilitazione automatica. 

### **Attivare il servizio Signal Hub in produzione**

L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `e_service.response.stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json).  

### **Portare in produzione il Credential Offer (opzionale)**

L'Ente, se precedentemente sviluppato in collaudo, deve provvedere in questo step al rilascio in produzione del Credential Offer all'interno delle proprie soluzioni.  

### **Notificare l'avvenuto rilascio in produzione**

> L'Ente deve notificare il Fornitore di Attestati di Attributi circa l'avvenuto rilascio dell'e-service in produzione su PDND, dei servizi di Signal Hub in produzione e, se previsto, del Credential Offer. In caso di EAA di interesse pubblico, l'Ente deve notificare IPZS inviando una mail all'indirizzo identitadigitale@pec.ipzs.it e, in caso di EAA disponibile nella soluzione pubblica di IT-Wallet, l'Ente deve notificare PagoPA inviando una mail all'indirizzo XX.

Reviewer 3: add email @silviafulci

1) Poniti le seguenti domande (puoi fare riferimento alla sezione “casi_d_uso”, già compilato):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
2) Formula un testo informativo rivolto all’utente a partire dai contenuti sopra raccolti, assicurandoti che sia chiaro, semplice, diretto e conciso (circa 300 - 450 caratteri, spazi inclusi).
## Appendice B – Data Model

---

## Template e-service PDND

Di seguito si condividono i template e-service PDND già definiti o contenuti nella soluzione pubblica IT-Wallet, che l'Ente dovrebbe prendere a riferimento per allineare il proprio e-service (sezione **e_service**) quando intende rendere disponibili dati appartenenti alle stesse categorie semantico-funzionali. In particolare: 

- EAA di affiliazione ad un ente o organizzazione 
- EAA di patenti o patentini 
- EAA di titoli o qualifiche 
- EAA di iscrizioni o frequenze

> 

Reviewer 3: ```suggestion

```

References not available. Section to be re-introduced further on

2. **campi opzionali** e **campi obbligatori**: nel caso l'EAA si riferisce a un documento fisico o digitale già esistente (es. patente) compila sia i campi obbligatori che i campi opzionali (`volume_rilascio`, `canali_richiesta`, `formato_attuale,` `pagamento`, ecc.). Nel caso invece l'EAA non abbia un documento preesistente corrispondente, compila solo i campi obbligatori e lascia vuoti i campi opzionali;
3. **risposta**: compila il campo `risposta` rispondendo in maniera chiara ed esaustiva ad ogni domanda; il campo `suggerimento` è solo indicativo.

**Esempio di compilazione** (fragmento):

```json
{
  "metadata": {
    "nome_ente_titolare": "Ministero dell'Istruzione",
    "nome_eaa": "Titolo di studio",
    "data_compilazione": "2026-03-16",
    "versione": "1.0"
  },
  "casi_d_uso": {
    "target_utenti": {
      "chi_puo_richiedere": {
        "domanda": "Chi può ad oggi richiedere il documento? ...",
        "suggerimento": "Solo maggiorenni residenti in una specifica regione",
        "risposta": "Tutti i cittadini italiani in possesso di diploma rilasciato da istituto italiano"
      }
    }
  }
}
```

> 

Author: ```suggestion
```
